### PR TITLE
Max mnc019

### PR DESCRIPTION
**Pull request by MassimoVilla**
One more Psalter from Casamari

### CODE REVIEW

File: Casamari/MNC019.xml [hunk 1]
```diff
@@ -0,0 +1,796 @@
...
+                                 <dim unit="leaf">10</dim>
+                                 <locus from="127r" to="136v"/>
+                              </item>
+                              <item xml:id="q15" n="14">
+                                 <dim unit="leaf">8</dim>
+                                 <locus from="137r" to="144v"/>
+                              </item>
+                           </list>
+                        </collation>
+                        <condition key="deficient">
```
Comment (SusanneHummel) [inline Casamari/MNC019.xml:576]: Do we all assign "deficient" in the case of restoration?
Added to our agenda

File: Casamari/MNC019.xml [hunk 2]
```diff
@@ -0,0 +1,796 @@
...
+                     </layoutDesc>
+                  </objectDesc>
+                  
+                  <handDesc>
+                     <handNote xml:id="h1" script="Ethiopic">
+                        <seg type="script"/>
+                        <seg type="ink">Black, red</seg>
+                        <date notBefore="1800" notAfter="1899">19th-century handwriting</date>
+                        <desc>
+                           Fine handwriting. Smaller in size in corrispondence of <ref target="#i1.4 #i1.5"/>.
```
Comment (SusanneHummel) [inline Casamari/MNC019.xml:635]: correspondence

File: Casamari/MNC019.xml [hunk 3]
```diff
@@ -0,0 +1,796 @@
...
+         </projectDesc>
+         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+            <xi:fallback>
+               <p>Definitions of prefixes used.</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc>
+         <particDesc>
+            <listPerson>
```
Comment (SusanneHummel) [inline Casamari/MNC019.xml:767]: I think I never provide this information. Is this important for the catalogue? Shall I also do it?

**Review by SusanneHummel — COMMENTED** [review]
Thank you Massimo!

File: Casamari/MNC019.xml [hunk 4]
```diff
@@ -0,0 +1,796 @@
...
+                  </msItem>
+               </msContents>
+          
+               <physDesc>
+                  <objectDesc form="Codex">
+                     <supportDesc>
+                        <support>
+                           <watermark/>
+                           <material key="parchment"/>
+                           <p>Holes, probably from parchment making, are found e.g. on <locus target="#3 #36 #78"/>.</p>
```
[the remark this answers is not in the document]
Comment (MassimoVilla) [reply Casamari/MNC019.xml:471]: removed

File: Casamari/MNC019.xml [hunk 5]
```diff
@@ -0,0 +1,796 @@
...
+                           <p>A parchment repair on <locus target="#74"/>.</p>
+                        </support>
+                        <extent>
+                           <measure unit="leaf">144</measure>
+                           <measure unit="leaf" type="blank">11</measure>
```
[the remark this answers is not in the document]
Comment (MassimoVilla) [reply Casamari/MNC019.xml:476]: Thank you. I think we can decide to leave the `<measure unit="leaf" type="blank"/>` information out in the printed catalogue (just like several other pieces of information which might make our descriptions too lengthy and space-consuming)

File: Casamari/MNC019.xml (hunk 1 above)
Comment (MassimoVilla) [reply Casamari/MNC019.xml:576]: My statement was way too severe! In reality the Ms is in good condition.

File: Casamari/MNC019.xml (hunk 3 above)
Comment (MassimoVilla) [reply Casamari/MNC019.xml:767]: In reality it is mostly redundant. An owner/scribe/donor/patron is nearly always indicated as such in a colophon, an additional note or a supplication formula. So, while entering or commenting those portions of text we can add `role="donor"` (or whatever, scribe, owner etc) inside persName (e.g. `<persName ref="PRS13014TaklaHa" role="scribe">ተክለ፡ ሐዋርያት።</persName>` from MNC014). In this way, all the relevant names can be extracted by the system and listed somewhere in the printed catalogue (e.g. in a prosopography section). The other cases, in which we know that a certain individual was involved in the history of the ms but (s)he is not mentioned in the Ms, can be anyway encoded under `<history><origin/></history>` or `<history><provenance/></history>`.
So, I frankly find `<listPerson/>` basically useless.

**Review by SusanneHummel — APPROVED** [review]

**Comment by MassimoVilla** [comment]
I am merging both this branch and MNC017. If further remarks arise around these two Psalters, I'll open a new branch and a new PR.

**Pull request merged**